proposals/008-topic-name-lookup-facility.md

```diff
@@ -0,0 +1,104 @@
+# Proposal 008: Resolve Topic Names from Topic IDs in the Filter Framework
+
+## Summary
+
+This proposal outlines a new feature for the Kroxylicious filter framework: a mechanism to resolve topic names from their corresponding topic IDs. This will enable filters to implement business logic based on human-readable topic names, rather than the underlying Kafka implementation detail of topic IDs.
+
```

Comment:
Might be worth referencing the KIP that gradually driving the introduction of topic-ids.